src/foam/nanos/u2/navigation/UserView.js

```diff
@@ -23,109 +23,190 @@ foam.CLASS({
+    ^ img{
+      width: 25px;
+      display: inline-block;
+      position: relative;
+      top: 2px;
+      right: 10px;
+      cursor: pointer;
+    }
+    ^user-name:hover {
+      cursor: pointer;
+    }
+    ^ .foam-nanos-menu-SubMenuView-inner {
+      position: absolute;
+      float: right;
+      z-index: 10001;
+      width: 215px;
+      background: white;
+      box-shadow: 2px 2px 2px 2px rgba(0, 0, 0, 0.19);
+      top: 65px;
+      right: 15px;
+    }
+    ^ .foam-nanos-menu-SubMenuView-inner > div {
+      height: 40px;
+      padding-left: 50px;
+      font-size: 14px;
+      font-weight: 300;
+      color: #093649;
+      line-height: 25px;
+    }
+    ^ .foam-nanos-menu-SubMenuView-inner > div:last-child {
+      background-color: #f6f9f9;
+      box-shadow: 0 -1px 0 0 #e9e9e9;
+      font-size: 14px;
+      color: #c82e2e;
+    }
+    ^ .foam-nanos-menu-SubMenuView-inner > div:hover {
+      background-color: %SECONDARYCOLOR%;
+      color: white;
+      cursor: pointer;
+    }
+    ^ .foam-nanos-menu-SubMenuView-inner::before {
+      content: ' ';
+      position: absolute;
+      height: 0;
+      width: 0;
+      border: 8px solid transparent;
+      border-bottom-color: white;
+      -ms-transform: translate(130px, -16px);
+      transform: translate(130px, -16px);
+    }
+    ^ .profile-container{
+      display: inline-block;
+      cursor: pointer;
+      padding-top: 10px;
+      height: 40px;
+    }
+    ^ img:hover {
+      cursor: pointer;
+      padding-bottom: 0px;
+      border-bottom: 1px solid white;
+    }
+    ^ img: {
+      height:30px;
+    }
+    ^ div .img {
+      width:30px;
+      display: inline-block;
+      //padding-bottom:0px;
+      cursor: pointer;
+      //border-bottom: 1px solid transparent;
+      -webkit-transition: all .15s ease-in-out;
+      -moz-transition: all .15s ease-in-out;
+      -ms-transition: all .15s ease-in-out;
+      -o-transition: all .15s ease-in-out;
+      transition: all .15s ease-in-out;
+    }
+    ^ .dot {
+      border-radius: 50%;
+      display: table-caption;
+      background: red;
+      width:15px;
+      height:15px;
+      position: relative;
+      top: -30px;
+      left:5px;
+      text-align: center;
+      font-size: 8px;
+    }
+    .net-nanopay-ui-topNavigation-UserTopNavView .dot > span {
+        padding-top: 3px;
+        display: inline-block;
+    }
+    /* ^ span {
+      top: -15px;
+    } */
+      .img{
+      display: inline-block;
+    }
+  `,
+
+  properties: [
+    {
+      class: "Int",
+      name: 'countUnread'
+    },
+    {
+      name: 'userCur',
+      factory: function(user){
+        return this.user;
+      }
+    }
   ],
+  
 
   methods: [
     function initE() {
+      this.notificationDAO.on.sub(this.onDAOUpdate);
+      this.user.id$.sub(this.onDAOUpdate);
+      this.onDAOUpdate();
+      var self = this;
       this
         .addClass(this.myClass())
+
+        .start('div').addClass('img').on('click', function() {
```

Comment:
I would move this whole block to its own NotificationIndicatorView or something. But you can do that if a subsequent PR.